lib/card/query/ref_spec.rb

```diff
@@ -1,9 +1,14 @@
 class Card::Query
   class RefSpec < Spec
     REFERENCE_DEFINITIONS = {
-      :refer_to => [ :out      ],   :referred_to_by => [ :in      ],
-      :link_to  => [ :out, 'L' ],   :linked_to_by   => [ :in, 'L' ],
-      :include  => [ :out, 'I' ],   :included_by    => [ :in, 'I' ]
+      # syntax:
+      # wql query key => [ direction, {reference_type} ]
+          # direction      = :out | :in
+          # reference_type =  'L' | 'I' | 'P' 
+
+      :refer_to => [ :out, 'L','I' ], :referred_to_by => [ :in, 'L','I' ],
+      :link_to  => [ :out, 'L' ],     :linked_to_by   => [ :in, 'L' ],
+      :include  => [ :out, 'I' ],     :included_by    => [ :in, 'I' ]
```

Comment:
It would simplify a few things to make the 'types' a string rather than an array of single characters.  Easy enough to iterate over the chars in a string (more than 1 in length.